doc/releases/release-notes-4.2.rst

```diff
@@ -33,19 +33,33 @@ We are pleased to announce the release of Zephyr version 4.2.0.
 
 Major enhancements with this release include:
 
-- Initial support for the **Renesas RX** 32-bit architecture has been added, including a QEMU-based
+**Initial Support for Renesas RX**
+  The Renesas RX architecture is now supported, including a QEMU-based
   :zephyr:board:`board target <qemu_rx>`.
 
-- The USB device stack now supports **USB Video Class** (UVC).
+**USB Video Class Driver**
+  The USB device stack now supports USB Video Class (UVC) allowing camera devices and other
+  image/video sources to be exposed as standard USB video devices. See :zephyr:code-sample:`uvc` to
+  get started.
 
-- The networking stack now includes full support for the **MQTT 5.0** protocol.
+**Twister Power Harness**
+  A :ref:`new Twister harness <twister_power_harness>` allows to measure the power consumption of
```

Comment:
"allows to measure" is grammatically incorrect. 

Recommend:

"enables measurement of"